genesis/process/sovereignGenesisBlockCreator.go

```diff
@@ -193,12 +192,6 @@ func createSovereignShardGenesisBlock(
 	indexingData.StakingTxs = stakingTxs
 
 	metaScrsTxs := metaProcessor.txCoordinator.GetAllCurrentUsedTxs(block.SmartContractResultBlock)
-	genesisESDTTransfers, err := createSovereignGenesisESDTTransfers(arg, shardProcessors.scrProcessor)
```

Comment:
Will you be able to see these transactions in the api/explorer? 
For the mainnet the genesis transactions had to be indexed after the beginning of the mainnet. 
They should be visible right from the start

Reply:
Transfers from `createSovereignGenesisESDTTransfers` are deleted.
The other ones, remain the same as in mainnet.

Reply:
At least generate LogEvents for the initial balance setups - than it should be good.

Reply:
Will add a task for it to be done on a separate PR